courseworks/week3/hw_3.md

```diff
@@ -0,0 +1,24 @@
+# 1
+
+$\Delta y_{rel} = \frac{a/(b+\epsilon)-a/b}{a/b}=\frac{\frac{1}{b+\epsilon}-\frac{1}{b}}{\frac{1}{b}}=\frac{-\epsilon}{b+\epsilon}
+$
+
+$\Delta x_{rel}=\frac{\epsilon}{b}$
+
+$\frac{\Delta y_{rel}}{\Delta _{rel}}=\frac{1}{1+\frac{\epsilon}{b}} $
+
+$\lim_{\epsilon \to 0} \frac{\Delta y_{rel}}{\Delta _{rel}} =1$ for any b given
+
+So the condition number of division operation is 1
```

Comment:
In your answer you assume that $a$ is a constant and $b$ is a variable and you got a correct result. However, the inverse case is also important, you can also check that.